src/haddock/modules/analysis/clustrmsd/defaults.yaml

```diff
@@ -42,8 +42,8 @@ min_population:
 clust_cutoff:
   default: 7.5
   type: float
-  min: 1
-  max: 9999
```

Comment:
can it be float and not have decimals here?

Reply:
yes!